examples/hellohttp/java/BUILD

```diff
@@ -33,7 +33,7 @@ k8s_deploy(
         "--v=2",
     ],
     images = {
-        "hello-http-image": ":server",
+        "hello-http-image:latest": ":server",
```

Comment:
why do you need to add this tag?

Reply:
At some point (I couldn't find where) when pushing images it tries to extract the image's tag from its name. When no colon is present in the name, it uses the default "latest" tag. When a colon is present, it extracts the part of the name that follows the last colon and considers that to be the tag. In the case when running tests locally and using a local registry in combination with image names without explicit tags, we end up with image names like "localhost:5000/myImage". In this case the extracted tag is "5000/myImage" and is invalid. Adding the default "latest" tag to where tags are not specified solves the problem.